src/modules/scraper/adapter/messaging/FeeBreakdownConsumer.ts

```diff
@@ -93,7 +99,7 @@ export class FeeBreakdownConsumer {
       relayCapitalFeePct: undefined,
       relayCapitalFeeAmount: undefined,
       relayGasFeeUsd,
-      relayGasFeePct: undefined,
+      relayGasFeePct: relayGasFeePct.toFixed(0),
```

Comment:
Do you plan to use `relayGasFeePct` in the FE instead of `relayGasFeeAmount`?

Reply:
Not sure yet... But I thought it would not hurt to add this